misc/services/camel/de-metas-camel-edi/src/main/java/de/metas/edi/esb/commons/DesadvSettings.java

```diff
@@ -49,18 +51,18 @@ public static DesadvSettings forReceiverGLN(
 		final String clearingCenterProperty = "edi.recipientGLN." + recipientGLN + ".clearingCenter";
 		if(Check.isBlank(recipientGLN))
 		{
-			return DesadvSettings.builder().clearingCenter(ClearingCenter.ecosio).build();
+			return DesadvSettings.builder().clearingCenter(ClearingCenter.MetasfreshInHouseV2).build();
 		}
 
-		final ClearingCenter clearingCenter = ClearingCenter.valueOf(Util.resolveProperty(context, clearingCenterProperty, "ecosio"));
+		final ClearingCenter clearingCenter = ClearingCenter.ofValue(Util.resolveProperty(context, clearingCenterProperty, MetasfreshInHouseV2.toString()));
```

Comment:
As written in the requirements 
- Currently one either has to set the clearingcenter in the properties-File for each GLN, or you have the hardcoded default of `MetasfreshInHouseV2`.
- what  we need in addition is way to deplare a **not-hardcoded** default in the properties-File

(same for `InvoicSettings.java`)